extensions/basic_auth/plugin.cc

```diff
@@ -227,6 +229,17 @@ bool PluginRootContext::configure(size_t configuration_size) {
                           configuration_data->view()));
     return false;
   }
+  auto it = j.find("realm");
+  if (it != j.end()) {
+    auto realm_string = JsonValueAs<std::string>(it.value());
+    if (realm_string.second != Wasm::Common::JsonParserResultDetail::OK) {
+      LOG_WARN(absl::StrCat(
+          "cannot parse realm in plugin configuration JSON string: ",
+          configuration_data->view()));
```

Comment:
Are we still returning false from here? We changed some other plugins to return true but log an error right?

Reply:
Yeah this is an auth filter, so I think fail close makes sense.